SPECS/symlinks/symlinks.spec

```diff
@@ -4,10 +4,10 @@ Vendor:         Microsoft Corporation
 Distribution:   Mariner
 URL: http://ibiblio.org/pub/Linux/utils/file/
 Version: 1.7
-Release: 3%{?dist}
+Release: 7%{?dist}
 License: Copyright only
 # Upstream maintainer provided tarball, ibiblio no longer allowing uploads
-Source0: http://ibiblio.org/pub/Linux/utils/file/%{name}-%{version}.tar.gz
+Source0: http://ibiblio.org/pub/Linux/utils/file/%{name}-%{version}.tar.gz#/%{name}-%{version}.tar.gz
```

Comment:
This URL seems to be dead. If that's the case, then we should do the following:
- Move the original URL to a comment
- Change Source0 to point to `%{_mariner_sources_url}/%{name}-%{version}.tar.gz`
- Update the `cgmanifest.json` file to point to our sources blobstore

Reply:
Done.